cosmos/cosmos-sdk
# Add 0.43 CLI JSON `migrate` command

## Description



closes: #8822 

To Test:
- run a 0.42 node for a couple of blocks
- stop the node, run `simd export > test.json`
- on this branch, run `simd migrate v0.43 test.json`

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.
